source/MaterialXRender/CMakeLists.txt

```diff
@@ -38,16 +38,11 @@ target_link_libraries(
 if(MATERIALX_BUILD_OIIO)
     set(OPENIMAGEIO_ROOT_DIR ${MATERIALX_OIIO_DIR})
     list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/External/OpenImageIO")
-    find_package(OpenImageIO REQUIRED)
-    if(OPENIMAGEIO_FOUND)
-        include_directories(${OPENIMAGEIO_INCLUDE_DIR})
-        target_link_libraries(${MATERIALX_MODULE_NAME} ${OPENIMAGEIO_LIBRARIES})
-        # Also needed by MaterialXTest:
-        set(OPENIMAGEIO_FOUND "${OPENIMAGEIO_FOUND}" PARENT_SCOPE)
-        set(OPENIMAGEIO_INCLUDE_DIR "${OPENIMAGEIO_INCLUDE_DIR}" PARENT_SCOPE)
-        set(OPENIMAGEIO_LIBRARIES "${OPENIMAGEIO_LIBRARIES}" PARENT_SCOPE)
-        set(OPENIMAGEIO_LIBRARY_DIRS "${OPENIMAGEIO_LIBRARY_DIRS}" PARENT_SCOPE)
-        set(OPENIMAGEIO_ROOT_DIR "${OPENIMAGEIO_ROOT_DIR}" PARENT_SCOPE)
+    find_package(OpenImageIO CONFIG REQUIRED)
+    if(OpenImageIO_FOUND)
+        target_link_libraries(${MATERIALX_MODULE_NAME} OpenImageIO::OpenImageIO OpenImageIO::OpenImageIO_Util)
+        # Also needed by MaterialXView:
+        set(OPENIMAGEIO_ROOT_DIR "${OpenImageIO_INCLUDE_DIR}/../" PARENT_SCOPE)
```

Comment:
Perhaps not completely related to this PR - but this line got me intrigued as to what `MaterialXView` was using `OPENIMAGEIO_ROOT_DIR` for, and it appears to be copying the binaries from OIIO in to the local cmake build folder.  I'm not exactly sure why this is, but if it's not necessary to do this anymore, we could clean up this line too.

If we need to keep it, then I might vote to move the `find_package()` call to a higher level CMakeLists.txt file, instead of using `PARENT_SCOPE` here, but thats more of a style choice than anything else.

Reply:
I agree, should I give it a try for review?

Reply:
Per @jstone-lucasfilm comment in slack - I would say yes. It's an easy enough thing to revert if it causes people issues.

Reply:
Agreed that this would be a great improvement, though I think it's fine to consider that as an orthogonal change to the current PR.